Ship_Game/GameScreens/ColonyScreen/ColonyScreen_Update.cs

```diff
@@ -82,6 +84,55 @@ void UpdateTradeTab()
             ManualExportTitle.Color  = GetManualExportSlotsOverrideColor();
         }
 
+        void UpdateDysonSwarmTab()
+        {
+            if (!IsDysonSwarmTabSelected)
+            {
+                if (DysonSwarmTypeTitle?.Visible == true)
+                    HideDysonSwarmUI();
+
+                return;
+            }
+            DysonSwarmTypeTitle.Visible       =
+            DysonSwarmControllerPanel.Visible =
+            DysonSwarmProdBoost.Visible       =
+            DysonSwarmPanel.Visible           = true;
```

Comment:
I understand that everything should be true here, but I think true on each row would be better